lib/macho/macho_file.rb

```diff
@@ -420,13 +420,13 @@ def delete_rpath(path)
     # @return [Array<MachO::Section64>] if the Mach-O is 64-bit
     def sections(segment)
       section_klass = case segment
-      when MachO::SegmentCommand
-        MachO::Section
-      when MachO::SegmentCommand64
-        MachO::Section64
-      else
-        raise ArgumentError, "not a valid segment"
-      end
+                      when MachO::SegmentCommand
+                        MachO::Section
+                      when MachO::SegmentCommand64
+                        MachO::Section64
+                      else
+                        raise ArgumentError, "not a valid segment"
+                      end
```

Comment:
If this looks unappealing to you, we could steal the following snippet from Homebrew's `.rubocop.yml` to gain a bit more flexibility with these:

```yaml
# we prefer compact if-else-end/case-when-end alignment
Lint/EndAlignment:
  AlignWith: variable
Style/CaseIndentation:
  IndentWhenRelativeTo: end
```